apps/members/app/src/components/Tables/AragonSortableTable.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import {Table, TableRow} from '@aragon/ui'
+import PropTypes from 'prop-types'
+import _ from 'lodash'
+import memoize from 'memoize-one'
+
+import HeaderTableCell, { SortType } from './HeaderTableCell'
+
+class AragonSortableTable extends React.Component {
```

Comment:
Why Aragon SortableTable (why Aragon in the name)?

Reply:
The table uses Aragon-UI for styling